<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Players</class>
 <widget class="QMainWindow" name="Players">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>389</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 2px;
}</string>
       </property>
       <property name="text">
        <string>Player list</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget {
background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);
border-radius: 2px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border-radius: 2px;
background-color: rgb(85, 85, 255);
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Filters</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Perpetua Titling MT</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
color: rgb(255, 255, 255);
background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
             </property>
             <property name="text">
              <string>Position</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Perpetua Titling MT</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
color: rgb(255, 255, 255);
background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
             </property>
             <property name="text">
              <string>Age</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Perpetua Titling MT</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
color: rgb(255, 255, 255);
background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
             </property>
             <property name="text">
              <string>Transfer value</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="forward_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>Forward</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mdf_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>Midfielder</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="def_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>Defender</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="gk_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>Goalkeeper</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="minus30_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>&lt; 30 y.o.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="plus30_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>&gt;= 30 y.o.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="minus25_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>&lt; 25 m.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="plus25_box">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
padding: 5px 10px;
}</string>
               </property>
               <property name="text">
                <string>&gt;= 25 m.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="filtr_button">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="info_button">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Short info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="site_button">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>More info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <family>Californian FB</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
